pkg/tests/ossm/smoke_test.go

```diff
@@ -15,55 +15,40 @@
 package ossm
 
 import (
-	"strings"
+	"net/http"
 	"testing"
-	"time"
 
 	"github.com/maistra/maistra-test-tool/pkg/app"
-	. "github.com/maistra/maistra-test-tool/pkg/examples"
-	. "github.com/maistra/maistra-test-tool/pkg/util"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/require"
+	"github.com/maistra/maistra-test-tool/pkg/util/curl"
 	. "github.com/maistra/maistra-test-tool/pkg/util/log"
-	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	. "github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupBookinfo() {
-	Log.Info("Cleanup")
-	app := Bookinfo{"bookinfo"}
-	app.Uninstall()
-	time.Sleep(time.Duration(30) * time.Second)
-}
-
 func TestBookinfo(t *testing.T) {
```

Comment:
The previous version of this test didn't just check if bookinfo returns a 200 OK, but also whether the pods have a sidecar, so let's rename it to express that fact.
```suggestion
func TestBookinfoInjection(t *testing.T) {
```